test/index.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import * as pidFromPort from 'pid-from-port'
+import { portToPid } from 'pid-port';
```

Comment:
You could reduce the amount of changes needed in this file by:

```suggestion
import { portToPid as pidFromPort } from 'pid-port';
```